runtime/libia2/include/ia2.h

```diff
@@ -107,6 +107,9 @@
   "movq %r11, %rdx\n"                           \
   "movq %r10, %rcx\n"
 /* clang-format on */
+#else
+// TODO aarch64
```

Comment:
I think `ASSERT_PKRU` is the only thing `IA2_DEBUG` toggles so we should be able to `#error "unimplemented"` here. If not, then this is fine too.

Reply:
I'd prefer not to `#error`, since then `-DIA2_DEBUG=1` only works conditionally, but I could do a `#warning` if you prefer.

Reply:
That's fine, just leave it as a comment then

Reply:
Sorry in my last comment I didn't realize `#warning` would only warn in that specific config. Either `#warning` or comment is good with me.